compliance/aws/scp_audit/aws_scp_audit.pt

```diff
@@ -48,7 +48,7 @@ pagination "pagination_aws_json" do
     body_path jmes_path(response, "NextToken")
   end
   set_page_marker do
-    query "NextToken"
+    body_field "NextToken"
```

Comment:
Looks good! I just noticed below that there is the following in the `ds_org_accounts` datasource:

```ruby
body_field "empty", "empy"
```

That's a little hacky, so while you are here, I think you could change it to an empty JSON object with:

```ruby
body "{}"
```

Reply:
You learn something new every day. Thanks @douglaswth!

Reply:
🤣 I actually implemented it two years ago so I was surprised to see that there were workarounds for the lack of it, but then I checked the docs and apparently it never made it in there, so now I'm going to add it there as well.